contrib/util/ccda_import/import_ccda.php

```diff
@@ -35,7 +35,7 @@
  */
 
 // comment this out when using this script (and then uncomment it again when done using script)
-exit;
+//exit;
```

Comment:
should keep the exit here (a safety measure so it can't be run until user comments this out since it will be a rarely used script).

Reply:
think it was reverted in the misnamed `comment` commit, should've been named `uncomment`